### ISSUE TITLE
v2.1: When uploading non-image media files to a sub-site, the sub-site ID is repeated twice in the URL, causing broken URLs.

### ISSUE THREAD

**Issue by CurtisBelt** [deliciousbrains/wp-amazon-s3-and-cloudfront#489]
Bug started in 2.1 -- reverted to 2.0 solved the problem. Probably related to the changes involved with:

> Bug fix: Year/Month path prefix incorrectly set in bucket for non-image media files

When I upload a PDF to a subsite, the file URL as far as WordPress is concerned is:

https://my-subsite.example.com/wp-content/uploads/sites/1566/2019/03/dummy1.pdf

Which is correct (I reverse-proxy /wp-content/uploads/* to the corresponding s3 URL).

But in the s3 bucket it's uploaded to:

s3://_example-bucket_/wordpress-uploads/_example-client_/wp-content/uploads/sites/1566/**1566**/2019/03/dummy1.pdf

As you can see the blog ID is duplicated. If I downgrade plugin to 2.0 the file correctly uploads to:

s3://_example-bucket_/wordpress-uploads/_example-client_/wp-content/uploads/sites/1566/2019/03/dummy1.pdf

All config settings are hardcoded in wp-config.php:

```php
define('AS3CF_AWS_USE_EC2_IAM_ROLE', true);
define('AS3CF_SETTINGS', serialize(array(
    'bucket' => $_SERVER['S3_UPLOADS_BUCKET'],
    'copy-to-s3' => true,
    'domain' => 'path',
    'enable-object-prefix' => true,
    'force-https' => false,
    'manual_bucket' => true,
    'object-prefix' => 'wordpress-uploads/' . $_SERVER['UPLOADS_S3_OBJECT_PREFIX'],
    'object-versioning' => false,
    'provider' => 'aws',
    'region' => '',
    'remove-local-file' => false,
    'serve-from-s3' => false,
    'use-yearmonth-folders' => true,
)));
```

**Comment by ianmjones** [comment]
Thanks @CurtisBelt, I can reproduce the problem once subsite IDs reach 4 digits and hope to have a fix in an upcoming release.

**Comment by CurtisBelt** [comment]
@ianmjones I'm doing fine on 2.0.1 for now, but just thought I'd check in here. Any word on next release? Thanks for checking into this.

**Comment by ianmjones** [comment]
@CurtisBelt The fix should be in v2.1.1 whenever that gets released! 😉

**Comment by ianmjones** [comment]
WP Offload Media Lite 2.1.1 has been released with a fix for this.
